src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -125,15 +126,34 @@ object DatabaseConversions extends LazyLogging {
     }
 
   implicit val blockAccountsToAccountHistoryRows =
-    new Conversion[List, BlockTagged[Map[AccountId, Account]], Tables.AccountsHistoryRow] {
-      override def convert(from: BlockTagged[Map[AccountId, Account]]): List[Tables.AccountsHistoryRow] =
-        blockAccountsToAccountRows.convert(from).map {
-          _.into[Tables.AccountsHistoryRow]
-            .withFieldConst(_.asof, Timestamp.from(from.timestamp.getOrElse(Instant.ofEpochMilli(0))))
-            .withFieldConst(_.cycle, from.cycle)
-            .withFieldConst(_.isBakerDeactivated, None)
+    new Conversion[List, (BlockTagged[Map[AccountId, Account]], List[AccountsHistoryRow]), Tables.AccountsHistoryRow] {
+      override def convert(
+          from: (BlockTagged[Map[AccountId, Account]], List[AccountsHistoryRow])
+      ): List[Tables.AccountsHistoryRow] = {
+        val (blockTaggedAccounts, accountsHistoryRowsWithInactiveBakers) = from
+        val touchedAccounts = blockAccountsToAccountRows.convert(blockTaggedAccounts).map { accountsRow =>
```

Comment:
Why are we not using the conversion syntax here, instead of `blockAccountsTo....convert(...)`?

Isn't that easier to read in `blockTaggedAccounts.convertToA[List, AccountsRow]`?

Reply:
thought about avoiding importing syntax, but sure, changed